logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -672,7 +633,7 @@ class TestPipeline < LogStash::Pipeline
         sleep(0.1) until pipeline.ready?
       end
       Stud.try(max_retry.times, [StandardError, RSpec::Expectations::ExpectationNotMetError]) do
-        wait(10).for do
+        wait(1).for do
```

Comment:
@andrewvc I think I just had some bug here in my testing and wanted to iterate quicker, reverting :)